department-of-veterans-affairs/abd-vro
# Add Github action for running Gradle release

## What was the problem?

Updating the app versions in code was a manual process 

Associated tickets or Slack threads:

- https://github.com/department-of-veterans-affairs/abd-vro/issues/1574

## How does this fix it?[^1]

Creates a new GitHub action that runs manually to update the code versions within the repo, push the new version tag, and any commits made from the Gradle release plugin.

## How to test this PR
- Run GH action 

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
